.changeset/rude-baboons-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/coinmetrics-adapter': major
+---
+
+add assets to coinmetrics EA asset-pairs endpoint
```

Comment:
Should only need one changeset file, and I'd expect this to be a `minor` change rather than `major` as it is backwards compatible: https://semver.org/